@claude - some of the unit tests are failing:
```
FAILED tests/unit/test_common.py::TestBuildUrl::test_simple_url - TypeError: quote_from_bytes() expected bytes
FAILED tests/unit/test_common.py::TestBuildUrl::test_url_with_none_params - TypeError: quote_from_bytes() expected bytes
FAILED tests/unit/test_common.py::TestBuildUrl::test_url_with_multiple_params - TypeError: quote_from_bytes() expected bytes
```

Could you please take a look and see about resolving these?

--- a/alpaca_pyspark/common.py
+++ b/alpaca_pyspark/common.py
@@ -77,8 +77,8 @@ def build_url(endpoint: str, path_elements: List[str], params: Dict[str, str]) -
     param_pairs = []
     for k, v in params.items():
         if v is not None:
-            # URL encode the value
-            quoted_v = urlp.quote(v)
+            # Convert to string and URL encode the value
+            quoted_v = urlp.quote(str(v))
             param_pairs.append(f"{k}={quoted_v}")
 
     param_str = "&".join(param_pairs)

